app/styles/_feeds.scss

```diff
@@ -82,37 +52,46 @@
 		box-shadow: 0;
 		border-radius: 0;
 		text-align: center;
-		
 	}
-
 }
+
 .success {
-	/*left: 0;*/
-    position: absolute;
-    text-align: center;
-    width: 100%;
-    /*margin-top: 50px;*/
+  position: absolute;
+  text-align: center;
+  width: 100%;
 }
 
 
-.input-form {
+.input-form-wrapper {
 	font-family: $base-font-family;
 	font-size: 16px;
 	color: #333333;
-	font-weight: normal;	
+	font-weight: normal;
 	line-height: $base-line-height;
-	background-color: #ffffff;
-	border-top: 2px solid $grey-color-light;
-	border-left: 2px solid $grey-color-light;
+	border: 2px solid $grey-color-light;
 
-	.input-section {
-		border-right: 2px solid $grey-color-light;
+	//this is to have row inside of this wrapper
+	padding-left: 15px;
+	padding-right: 15px;
 
-	}
+	margin-bottom: 20px;
 
-	.row {
+	//Put bottom border for row inside of input-form-wrapper unless it is last child element
+	.row:not(:last-child) {
```

Comment:
and row class inside of `input-form-wrapper` has border bottom, unless it is last child.